pkg/kubelet/metrics/metrics.go

```diff
@@ -33,6 +33,9 @@ const (
 	KubeletSubsystem                   = "kubelet"
 	NodeNameKey                        = "node_name"
 	NodeLabelKey                       = "node"
+	KubeletBootstrapKey                = "node_start_from_kubelet_running_to_registration_attempt_duration_seconds"
+	NodeRegistrationKey                = "node_start_from_registration_attempt_to_registered_duration_seconds"
+	NodeReadyKey                       = "node_start_from_registered_to_node_ready_duration_seconds"
```

Comment:
```suggestion
	KubeletBootstrapKey                = "node_bootstrap_to_register_duration_seconds"
	NodeRegistrationKey                = "node_register_duration_seconds"
	NodeReadyKey                       = "node_ready_duration_seconds"
```
The name is too long.

Another way to do this is like `SLI Metrics`.

```
# HELP kubelet_node_start_duration_seconds [ALPHA] This metric records the result of node startup duration from running to registration attempt, from registration attempt to registered, from registered to ready.
# TYPE kubelet_node_start_duration_seconds gauge
kubelet_node_start_duration_seconds{name="running_to_registration_attempt"} 0.22s
kubelet_node_start_duration_seconds{name="registration_attempt_to_registered"} 0.02s
kubelet_node_start_duration_seconds{name="registered_to_ready"} 3.122s
kubelet_node_start_duration_seconds{name="all"} 3.362s
```

Reply:
That looks better. Let me see how to do that

Reply:
Updated the metrics definition. See if it is aligned with your idea. @pacoxu